jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/PrivateMessageControllerTest.java

```diff
@@ -437,6 +411,20 @@ private PrivateMessage getPrivateMessage() {
                 new JCUser("username2", "email2", "password2"), "title", "body");
     }
 
+    /**
+     * @param hasGlobalError true if we have GlobalError like an ObjectError.
+     * @param errorFields list of fields witch have validation errors can be
+     *                    empty if we don't have any.
+     * @return MockMvc instance for unit testing.
+     */
+    private MockMvc getMockMvcWithValidator(boolean hasGlobalError, String... errorFields){
+        Validator validator = new ValidatorStub(errorFields);
+        if (hasGlobalError){
```

Comment:
Maybe you did that because the last time I asked to factor out some logic into readable method. I meant this logic: 
```
mockMvc.perform(post("/pm/save")
                .param("id", "1")
                .param("title", "")
                .param("body", "")
                .param("recipient", ""))```
Can be factored out into method `Response saveDraft(Draft draft) {}` or `Response saveDraft(title, body, recepient) {}`